docs/specviz/notebook.rst

```diff
@@ -1,5 +1,20 @@
 ***********************************
-Using SpecViz in a Jupyter Notebook 
+Using Specviz in a Jupyter Notebook 
 ***********************************
 
-Words go here...
+Specviz is developed to fully support analyzing spectra within your existing `Jupyter notebooks <https://jupyter.org/>`_! To use Specviz, install jdaviz in your notebook's python environment and add a new cell wherever you would like to use Specviz.
+::
+
+    # Import specviz
```

Comment:
If you want to use code segments here, you can use sphinx syntax that is on the bottom of this page as a template: https://github.com/spacetelescope/jdaviz/blob/master/docs/specviz/import_data.rst